backend/Dockerfile.backend.new

```
@@ -0,0 +1,59 @@
+ENV PYTHONUNBUFFERED=1
+# Store poetry cache in image
+ENV POETRY_CACHE_DIR=/root/.cache/pypoetry
+# No need to set PYTHONPATH maybe
+# because we set WORKDIR
+
+WORKDIR /backend
+
+# Install FFmpeg and curl
+# Remove cache to reduce image size
+# https://genzouw.com/entry/2019/09/04/085135/1718/
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)